content/docs/searching.md

```diff
@@ -70,8 +70,21 @@ The various action buttons available include:
 * **Close**: Closes the search dialog
 
 * **Replace**: Replaces the currently-selected match.  (If no match is currently selected, it behaves like **Find Next** and just highlights the next match in the specified direction)
-* **Replace All**: Replaces all matches from the active location onward (following the **Backward direction** and **Wrap around** settings as appropriate).
+* **Replace All**: Replaces all matches in the active document.  With **☑ Wrap Around** enabled, it makes one pass through the active document, from the very top to the very bottom.  With **☐ Wrap Around** disabled, it searches from the caret to the end of the file (if **☐ Backward direction** is disabled) or from the caret to the beginning of the file (if **☑ Backward direction** is enabled).
```

Comment:
It leads off with "Replaces all matches in the active document".  But then goes on to explain how that may not be true.

Reply:
Removed that sentence and tweaked the next two sentences to be more accurate.